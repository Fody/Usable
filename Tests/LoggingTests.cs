using System.Linq;
using System.Runtime.CompilerServices;
using ApprovalTests.Reporters;
using NUnit.Framework;

[TestFixture]
[UseReporter(typeof(DiffReporter))]
public class LoggingTests
{
    [Test]
    [MethodImpl(MethodImplOptions.NoInlining)]
    public void InfoMessages()
    {
        ApprovalTests.Approvals.VerifyAll(AssemblyWeaver.Infos.OrderBy(s => s), "Info");
    }

    [Test]
    [MethodImpl(MethodImplOptions.NoInlining)]
    public void WarningMessages()
    {
        ApprovalTests.Approvals.VerifyAll(AssemblyWeaver.Warnings.OrderBy(s => s), "Warning");
    }

    [Test]
    [MethodImpl(MethodImplOptions.NoInlining)]
    public void ErrorMessages()
    {
        ApprovalTests.Approvals.VerifyAll(AssemblyWeaver.Errors.OrderBy(s => s), "Error");
    }
}