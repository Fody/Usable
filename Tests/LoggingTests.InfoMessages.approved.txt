Info[0] = Async method 'System.Threading.Tasks.Task BrokenScenarios::Issue7()' cannot be modified by Usable.
